.github/workflows/release-drafter.yml

```diff
@@ -9,7 +9,27 @@ jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      - name: Draft release notes
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          # Only need package.json as it is used by the get_version step to pass tag number into release-drafter
+          sparse-checkout: package.json
+
+      - name: Get version number from package.json
+        id: get_version
+        shell: bash
+        run: |
+          set -e
+          echo "version_number=$(jq -r '.version' package.json)" >> $GITHUB_OUTPUT
```

Comment:
Sorry, I'm a bit rusty on GH actions, and they changed how to do things in the past. Is this the recommended way to store some values? Can you point me to some docs?

Reply:
@bartaz Yes, this is the "new" way to set an output param for a GHA step. See some docs here: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

You may have seen this in the past with a slightly different syntax that was deprecated recently; maybe this looks more familiar: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/